locale/el_GR/default.po

```diff
@@ -145,10 +145,10 @@ msgstr ""
 "\tΗ παράγραφος που ξεκινάει \"Αυτό το τελευταίο θέμα...\" δεν έχει εσοχή.</pre>"
 
 msgid "default.journalSettings.forReaders"
-msgstr "Ενθαρρύνουμε τους αναγνώστες να εγγραφούν στην υπηρεσία αποστολής ενημερώσεων του περιοδικού. Χρησιμοποιήστε τον σύνδεσμο &lt;a href=\"{$indexUrl}/{$journalPath}/user/register\"&gt;Εγγραφή&lt;/a&gt; στην κορυφή της αρχικής σελίδας του περιοδικού. Οι εγγεγραμμένοι Χρήστες λαμβάνουν ηλεκτρονικά τον πίνακα περιεχομένων κάθε νέου τεύχους του περιοδικού. Επίσης, η λίστα των εγγεγραμμένων χρηστών-αναγνωστών επιτρέπει στην ομάδα διαχείρισης του περιοδικού να διαπιστώσει το πραγματικό επίπεδο χρήσης του. Πριν την εγγραφή σας, διαβάστε την &lt;a href=\"{$indexUrl}/{$journalPath}/about/submissions#privacyStatement\"&gt;Πολιτική Προστασίας Προσωπικών Δεδομένων&lt;/a&gt; του περιοδικού"
+msgstr "Ενθαρρύνουμε τους αναγνώστες να εγγραφούν στην υπηρεσία αποστολής ενημερώσεων του περιοδικού. Χρησιμοποιήστε τον σύνδεσμο &lt;a href=\"{$indexUrl}{$contextPath}}/user/register\"&gt;Εγγραφή&lt;/a&gt; στην κορυφή της αρχικής σελίδας του περιοδικού. Οι εγγεγραμμένοι Χρήστες λαμβάνουν ηλεκτρονικά τον πίνακα περιεχομένων κάθε νέου τεύχους του περιοδικού. Επίσης, η λίστα των εγγεγραμμένων χρηστών-αναγνωστών επιτρέπει στην ομάδα διαχείρισης του περιοδικού να διαπιστώσει το πραγματικό επίπεδο χρήσης του. Πριν την εγγραφή σας, διαβάστε την &lt;a href=\"{$indexUrl}{$contextPath}}/about/submissions#privacyStatement\"&gt;Πολιτική Προστασίας Προσωπικών Δεδομένων&lt;/a&gt; του περιοδικού"
 
 msgid "default.journalSettings.forAuthors"
-msgstr "Εάν ενδιαφέρεστε να υποβάλετε την εργασία σας στο συγκεκριμένο περιοδικό, ανατρέξετε στη σελίδα &lt;a href=\"{$indexUrl}/{$journalPath}/about\"&gt;Σχετικά με το περιοδικό&lt;/a&gt; για τις πολιτικές κάθε ενότητας του περιοδικού, καθώς και στη σελίδα &lt;a href=\"{$indexUrl}/{$journalPath}/about/submissions#authorGuidelines\"&gt;Οδηγίες προς τους Συγγραφείς&lt;/a&gt;. Οι Συγγραφείς πρέπει να κάνουν &lt;a href=\"{$indexUrl}/{$journalPath}/user/register\"&gt;εγγραφή&lt;/a&gt; στο περιοδικό πριν από την υποβολή ή, σε περίπτωση που έχουν ήδη εγγραφεί, μπορούν να κάνουν &lt;a href=\"{$indexUrl}/{$journalPath}/login\"&gt;Σύνδεση&lt;/a&gt; και να ξεκινήσουν τη διαδικασία υποβολής σε 5 βήματα."
+msgstr "Εάν ενδιαφέρεστε να υποβάλετε την εργασία σας στο συγκεκριμένο περιοδικό, ανατρέξετε στη σελίδα &lt;a href=\"{$indexUrl}{$contextPath}}/about\"&gt;Σχετικά με το περιοδικό&lt;/a&gt; για τις πολιτικές κάθε ενότητας του περιοδικού, καθώς και στη σελίδα &lt;a href=\"{$indexUrl}{$contextPath}}/about/submissions#authorGuidelines\"&gt;Οδηγίες προς τους Συγγραφείς&lt;/a&gt;. Οι Συγγραφείς πρέπει να κάνουν &lt;a href=\"{$indexUrl}{$contextPath}}/user/register\"&gt;εγγραφή&lt;/a&gt; στο περιοδικό πριν από την υποβολή ή, σε περίπτωση που έχουν ήδη εγγραφεί, μπορούν να κάνουν &lt;a href=\"{$indexUrl}{$contextPath}}/login\"&gt;Σύνδεση&lt;/a&gt; και να ξεκινήσουν τη διαδικασία υποβολής σε 5 βήματα."
```

Comment:
I think this introduces a typo (see the `}}` close).

Reply:
Fixed. Thank!

Reply:
Is it `{$indexUrl}{$contextPath}` outdated? I don't know if `{$indexUrl}/{$journalPath}` is the true way, because I noticed that in the Spanish version the webpage states that it doesn't have a context when I tried to access the default information for authors or readers.

Reply:
Thanks, @Mandroide! It should definitely be `$contextPath` rather than `$journalPath`, and the `$indexUrl` should include a trailing slash (at least on my dev system, which is pretty typical). Does the URL it's generating appear to include a typo?

Reply:
Neither {$indexUrl}{$contextPath} nor {$indexUrl}/{$journalPath} seem to include a typo.

Reply:
I'm not sure I'm following what you said earlier...
> In the Spanish version the webpage states that it doesn't have a context when I tried to access the default information for authors or readers.

Does the link to the Spanish URL look different from the link to the English URL?

Reply:
Oh I see. In the English Version `{$indexUrl}{$contextPath}` shows a Not Found on the server error, while in the Canada French Version `{$indexUrl}/{$journalPath}` the generated URL is working. 

I don't understand why in the Spanish version appears that it cannot find a context, if the way it generates the URL is the same as Canade French Version.